routes/web.php

```diff
@@ -19,3 +20,9 @@
 Route::get('/contacts', [MainController::class, 'contacts'])->name('contacts');
 
 Route::get('/about', [MainController::class, 'about'])->name('about');
+
+//Route::get('/report', [ReportController::class, 'show'])->name('report');
+
+Route::post('/report', [ReportController::class, 'store'])->name('report_store');
```

Comment:
у нас же речь про контакты, а не про жалобы
```suggestion
Route::post('/contacts', [ContactController::class, 'store'])->name('contact_store');
```